samples/maps.go

```diff
@@ -0,0 +1,28 @@
+// Maps
+package main
+
+import "fmt"
+
+func main() {
+	// A map maps keys to values.
+	// Create an empty map with builtin `make`:
+	m1 := make(map[string]int)
```

Comment:
I'm not a fan of make(), generally only using when it's the only option. I much prefer `map[string]int{}`. Not sure I'm in the majority though.

Reply:
Agreed - Is there any reason beyond preallocation to use it with `map`s?

Reply:
disagree. there is an example further down of a map literal. the sample should show both.

@julia, i suggest speaker notes at this point.

Reply:
The example below shows a non-empty literal, which might give the impression that empty maps should be created with make(). I don't want anyone coming away with that impression. I would be happy for the example to supply a capacity parameter, which is an unarguably valid use for make().

Reply:
Ok, I've added the sample further down, @anzdaddy.

Reply:
I can live with that.

Reply:
I can live with that.